src/main/java/uk/gov/hmcts/juror/api/moj/report/grouped/TrialAttendanceReport.java

```diff
@@ -0,0 +1,124 @@
+            ReportsJurorPaymentsDataTypes.PAYMENT_AUDIT,
+            ReportsJurorPaymentsDataTypes.TOTAL_DUE,
+            ReportsJurorPaymentsDataTypes.TOTAL_PAID);
+
+        this.courtLocationRepository = courtLocationRepository;
+        this.trialRepository = trialRepository;
+
+        isCourtUserOnly();
+    }
+
+    @Override
+    public Class<? extends Validators.AbstractRequestValidator> getRequestValidatorClass() {
+        return RequestValidator.class;
+    }
+
+    @Override
+    protected void preProcessQuery(JPAQuery<Tuple> query, StandardReportRequest request) {
+        query.where(QReportsJurorPayments.reportsJurorPayments.trialNumber.eq(request.getTrialNumber()));
+        query.orderBy(QReportsJurorPayments.reportsJurorPayments.jurorNumber.asc());
+    }
+
+    @Override
+    public Map<String, GroupedReportResponse.DataTypeValue> getHeadings(
+        StandardReportRequest request,
+        StandardReportResponse.TableData<GroupedTableData> tableData) {
+
+        Optional<Trial> optTrial = trialRepository.findByTrialNumberAndCourtLocationLocCode(request.getTrialNumber(),
+                                                                                         SecurityUtil.getLocCode());
+
+        Trial trial = optTrial.orElse(null);
+        assert trial != null;
+
+        Map<String, GroupedReportResponse.DataTypeValue> map = new ConcurrentHashMap<>();
+        map.put("trial_number", GroupedReportResponse.DataTypeValue.builder()
+            .displayName("Trial number")
+            .dataType("String")
+            .value(request.getTrialNumber())
+            .build());
+        map.put("trial_names", GroupedReportResponse.DataTypeValue.builder()
+            .displayName("Names")
+            .dataType("String")
+            .value(trial.getDescription())
+            .build());
+        map.put("trial_type", GroupedReportResponse.DataTypeValue.builder()
+            .displayName("Trial type")
+            .dataType("String")
+            .value(trial.getTrialType().getDescription())
+            .build());
+        map.put("trial_start_date", GroupedReportResponse.DataTypeValue.builder()
+            .displayName("Trial start date")
+            .dataType("LocalDate")
+            .value(DateTimeFormatter.ISO_DATE.format(trial.getTrialStartDate()))
+            .build());
+        map.put("trial_courtroom", GroupedReportResponse.DataTypeValue.builder()
+            .displayName("Courtroom")
+            .dataType("String")
+            .value(trial.getCourtroom().getDescription())
+            .build());
+        map.put("trial_judge", GroupedReportResponse.DataTypeValue.builder()
+            .displayName("Judge")
+            .dataType("String")
+            .value(trial.getJudge().getName())
+            .build());
+
+        map.put("court_name", AbstractReportResponse.DataTypeValue.builder().displayName("Court Name")
+            .dataType(String.class.getSimpleName())
```

Comment:
Should be able to use loadStandardTrialHeaders(...)

Reply:
Fixed